doc/classes/HFlowContainer.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="HFlowContainer" inherits="FlowContainer" version="4.0">
+	<brief_description>
+		Horizontal flow container.
+	</brief_description>
+	<description>
+		Horizontal flow container. See [FlowContainer].
```

Comment:
```suggestion
		Horizontal version of [FlowContainer].
```
No need to repeat the brief description. (same below)

Reply:
Fixed for both container variants.